templates/encounter/forms/navbar.html.twig

```diff
@@ -16,7 +16,7 @@
                         <a class="nav-link dropdown-toggle" href="#" id="category_{{ category|attr }}" role="button" data-toggle="dropdown" aria-expanded="false">
                             {{ category|text }}
                         </a>
-                        <div class="dropdown-menu" aria-labelledby="category_{{ category|attr }}">
+                        <div class="dropdown-menu" aria-labelledby="category_{{ category|attr }}" style="z-index:9999;">
```

Comment:
Unsure if any other form would need to override this menu but next level in the z-index is 1050. Anyway this is fine as is.

Reply:
thanks!